opmlexport/opmlexport.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Name: OPML Export
+ * Description: Export user's RSS/Atom contacts as OPML
+ * Version: 1.0
+ * Author: Fabio Comuni <https://social.gl-como.it/profile/fabrixxm>
+ * License: 3-clause BSD license
+ */
+
+use Friendica\DI;
+use Friendica\App;
+use Friendica\Core\Hook;
+use Friendica\Core\Logger;
+use Friendica\Network\HTTPException;
+use Friendica\Database\DBA;
+use Friendica\Core\Renderer;
+
+function opmlexport_install() {
```

Comment:
Code standards: Please add a new line before function opening braces.
```suggestion
function opmlexport_install()
{
```